package.json

```diff
@@ -177,6 +177,7 @@
     "lint-staged": "^11.2.4",
     "np": "^7.5.0",
     "npm-run-all": "^4.1.5",
+    "postcss-html": "^1.1.0",
```

Comment:
[question] Can you bump the version to [1.2.0](https://github.com/ota-meshi/postcss-html/releases/tag/v1.2.0)?

Reply:
Done